guides/common/modules/proc_including-power-actions-in-templates.adoc

```diff
@@ -1,13 +1,15 @@
-:_mod-docs-content-type: REFERENCE
+:_mod-docs-content-type: PROCEDURE
 
 [id="Including_Power_Actions_in_Templates_{context}"]
 = Including power actions in templates
 
+[role="_abstract"]
 This example shows how to set up a job template for performing power actions, such as reboot.
 This procedure prevents {Project} from interpreting the disconnect exception upon reboot as an error, and consequently, remote execution of the job works correctly.
 
-Create a new template as described in xref:customizing-job-templates[], and specify the following string in the template editor:
-
+.Procedure
+* Create a new template as described in xref:customizing-job-templates[], and specify the following string in the template editor:
++
 [source, ruby]
 ----
 <%= render_template("Power Action - SSH Default", :action => "restart") %>
```

Comment:
Out of scope but wrong tech-wise: the type has been renamed from "SSH" to "Script". You can verify this in foreman_remote_execution:

````
$ rg "Power Action"
app/views/templates/script/power_action.erb
3:name: Power Action - Script Default
````

I can fix that as a follow-up.

Reply:
will do!